app/presenters/idv/by_mail/letter_enqueued_presenter.rb

```diff
@@ -24,21 +24,25 @@ def address_lines
       end
 
       def button_text
-        if sp
-          t('idv.cancel.actions.exit', app_name: APP_NAME)
-        else
-          t('idv.buttons.continue_plain')
-        end
+        t('idv.cancel.actions.exit', app_name: APP_NAME)
       end
 
       def button_destination
         if sp
           return_to_sp_cancel_path(step: :verify_address, location: :come_back_later)
         else
-          account_path
+          marketing_site_redirect_url
```

Comment:
Taking the user to their account page when they clicked "Exit Login.gov" was very random.

(Arguably, so is taking them to the static site homepage, but it's less-inexplicable.)

Reply:
FWIW we have [a big epic](https://cm-jira.usa.gov/browse/LG-11712) on the roadmap that is all about removing these `if sp then x else y` type constructions--in reality, _every_ IdV attempt should be associated with an SP.